locust_plugins/jmeter_listener.py

```diff
@@ -99,7 +103,7 @@ def _flush_to_log(self):
         self.results_file.flush()
         self.csv_results = []
 
-    def _write_final_log(self):
+    def _write_final_log(self, **kwargs):
```

Comment:
Why accept kwargs that is not used?

Reply:
Thanks - I had an idea and backed it out...

Reply:
Looked at it further. This method is using an event hook so needs to accommodate whatever is passed in.

Reply:
Ok! If pylint gives you attitude about unused parameter, you can name it _kwargs instead.

Reply:
without the unpacking operator?

Reply:
no, with. so **_kwargs
Sorry if I was unclear..

Reply:
You still have a couple more unused arguments, other than that it is looking good!